### ISSUE TITLE
feat: support application/x-www-form-urlencoded content-type

### ISSUE THREAD

**Issue by tdakkota** [ogen-go/ogen#198]
See https://spec.openapis.org/oas/v3.1.0#support-for-x-www-form-urlencoded-request-bodies

**Comment by tdakkota** [comment]
Fixed by #351 and #359

**Comment by yonesko** [comment]
Why am I still receiving erorr with 1.12.0:
```
Content type "x-www-form-urlencoded" is unsupported.

Try to create ogen.yml with:

generator:
        ignore_not_implemented: ["unsupported content types"]

```
